persist-cli/src/main/java/io/ballerina/persist/nodegenerator/TomlSyntaxGenerator.java

```diff
@@ -130,6 +134,35 @@ public static PersistConfiguration readPersistConfigurations(String schemaName,
         }
     }
 
+
+    public static String readPackageName(String sourcePath) throws BalException {
+        try {
+            TextDocument configDocument = TextDocuments.from(Files.readString(Paths.get(sourcePath, BALLERINA_TOML)));
+            SyntaxTree syntaxTree = SyntaxTree.from(configDocument);
+            DocumentNode rootNote = syntaxTree.rootNode();
+            NodeList<DocumentMemberDeclarationNode> nodeList = rootNote.members();
+            for (DocumentMemberDeclarationNode member : nodeList) {
+                if (member instanceof TableNode) {
+                    TableNode node = (TableNode) member;
+                    String tableName = node.identifier().toSourceCode().trim();
+                    if (tableName.equals(KEYWORD_PACKAGE)) {
+                        NodeList<KeyValueNode> fields = node.fields();
+                        for (KeyValueNode field : fields) {
+                            if (field.identifier().toSourceCode().trim()
+                                    .equals(KEYWORD_NAME)) {
+                                return field.value().toSourceCode().trim().replaceAll("\"", "");
+                            }
+                        }
+                    }
+
+                }
+            }
+            throw new BalException("Error while reading Ballerina.toml. Package name could not be read. ");
+        } catch (IOException e) {
+            throw new BalException("error while Ballerina.toml. " + e.getMessage());
```

Comment:
```suggestion
            throw new BalException("Couldn't read the Ballerina.toml file. " + e.getMessage());
```

Reply:
updated in [3d46762](https://github.com/ballerina-platform/persist-tools/pull/127/commits/3d467623d38eb6d6ce04ffb0efbaf105f1dafe7f)